cfme/tests/cloud/test_keypairs.py

```diff
@@ -220,7 +220,7 @@ def test_keypair_visibility_in_tenants(appliance, child_tenant_admin_user):
 
 
 @test_requirements.cloud
-@pytest.mark.meta(automates=[1713400])
+@pytest.mark.meta(automates=[1713400, 1747179])
```

Comment:
I am not sure why we have both, but I think the BZ should go to the docblock as well

Reply:
Added to BZ docblock.

Reply:
@jaryn old habits die hard. its optional, the meta mark is what matters for automation.

Reply:
We encourage it to be used to list BZ's that might be tangentially related to the case. They're not blockers, or automated coverage, but maybe they're important to the context of the test case, and history of the behavior in MIQ.